catalyst-uploader.go

```diff
@@ -40,7 +40,12 @@ func run() int {
 	timeout := fs.Duration("t", 30*time.Second, "Upload timeout")
 	storageFallbackURLs := CommaMapFlag(fs, "storage-fallback-urls", `Comma-separated map of primary to backup storage URLs. If a file fails uploading to one of the primary storages (detected by prefix), it will fallback to the corresponding backup URL after having the prefix replaced`)
 
-	_ = fs.String("config", "", "config file (optional)")
+	defaultConfigFile := "/etc/livepeer/catalyst_uploader.conf"
+	_, err = os.Stat(defaultConfigFile)
+	if err != nil && os.IsNotExist(err) {
```

Comment:
**nit:** I blieve `err != nil` is redundant, we could even 1-line this 👺
```suggestion
	if _, err := os.Stat(defaultConfigFile); os.IsNotExist(err) {
```

Reply:
Yeah thought you'd probably pick up on that :)